DESCRIPTION

```diff
@@ -1,19 +1,19 @@
 Package: estimatr
 Type: Package
-Title: Fast estimators for social scientists
-Version: 0.0.9
+Title: Fast Estimators for Social Scientists
+Version: 0.2.0
 Date: 2017-12-18
 Authors@R: c(person("Graeme", "Blair", email = "graeme.blair@ucla.edu", role = c("aut", "cre")),
              person("Jasper", "Cooper", email = "jjc2247@columbia.edu", role = c("aut")),
              person("Alexander", "Coppock", email = "alex.coppock@yale.edu", role = c("aut")),
              person("Macartan", "Humphreys", email = "macartan@gmail.com", role = c("aut")),
              person("Luke", "Sonnet", email = "luke.sonnet@gmail.com", role = c("ctb")),
-             person("Neal", "Fultz", email="nfultz@gmail.com", role = c("ctb")))
-Description: This package provides fast estimators for common models in social science research.
+             person("Neal", "Fultz", email = "nfultz@gmail.com", role = c("ctb")))
+Description: Quickly estimate common models in social science research with robust standard errors.
 URL: http://estimatr.declaredesign.org, https://github.com/DeclareDesign/estimatr
 BugReports: https://github.com/DeclareDesign/estimatr/issues
 License: MIT + file LICENSE
-Depends:	R (>= 2.10)
+Depends: R (>= 2.10)
```

Comment:
Might as well bump this to at least 3.3, that's our oldest supported version for this release cycle.